libs/vkd3d/swapchain.c

```diff
@@ -2658,39 +2666,28 @@ static HRESULT d3d12_swapchain_init(struct d3d12_swapchain *swapchain, IDXGIFact
         return hr;
     }
 
-    fence_desc.sType = VK_STRUCTURE_TYPE_FENCE_CREATE_INFO;
-    fence_desc.pNext = NULL;
-    fence_desc.flags = 0;
-    if ((vr = vk_procs->vkCreateFence(vk_device, &fence_desc, NULL, &vk_fence)) < 0)
-    {
-        WARN("Failed to create Vulkan fence, vr %d.\n", vr);
-        d3d12_swapchain_destroy(swapchain);
-        return hresult_from_vk_result(vr);
-    }
-    swapchain->vk_fence = vk_fence;
-
     swapchain->current_buffer_index = 0;
 
+    swapchain->frame_number = DXGI_MAX_SWAP_CHAIN_BUFFERS;
+    swapchain->frame_latency = 3;
+
     if (swapchain_desc->Flags & DXGI_SWAP_CHAIN_FLAG_FRAME_LATENCY_WAITABLE_OBJECT)
-    {
-        swapchain->frame_number = DXGI_MAX_SWAP_CHAIN_BUFFERS;
-        swapchain->frame_latency = 1;
+        swapchain->frame_latency = 3;
```

Comment:
You already set it to three, why are you doing it again?

Reply:
I made a typo.